CHANGELOG.md

```diff
@@ -18,6 +18,16 @@
   simply forwarded to the Firecracker executable using "end of command
   options" convention.
 - Decreased release binary size by ~15%.
+- Moved firecracker's default API socket path form `/tmp/firecracker.socket`
```

Comment:
We can just say "Changed the default API socket path to `/run/firecracker.socket`". It is implied that it is firecracker's API socket and we generally want to keep the changelog entries as short as possible.
I would personally use the same changelog entry to say that "This applies when running with the jailer as well".

And then I would have the fact that you can change the default API socket when running with the jailer in the `Fixed` section because it was sort of a bug.

What do you say?

Reply:
nice suggestion @andreeaflorescu , will change as like you suggest 👍

Reply:
@andreeaflorescu changed doc as like you suggested 🙂.